src/index.js

```diff
@@ -51,9 +53,18 @@ export default class HillChart extends EventEmitter {
     this.chartHeight = height - margin.top - margin.bottom;
 
     // Render the svg and center chart according to margins
+    const colorScheme = this.darkMode ? 'hill-chart-dark' : 'hill-chart-light';
+    const defaultBg = this.darkMode ? '#2f3437' : '#ffffff';
+    const useDefaultBg = this.backgroundColor === true;
+    const useTransparentBg = this.backgroundColor === false;
+    const suppliedBgColor = useDefaultBg ? defaultBg : this.backgroundColor;
+    const backgroundColor = useTransparentBg ? 'transparent' : suppliedBgColor;
```

Comment:
```suggestion
    this.backgroundColor = useTransparentBg ? 'transparent' : suppliedBgColor;
```
Same as `colorScheme`